src/test/java/com/Automation/APIAutomation/UserManagement/postUsers.java

```diff
@@ -82,7 +77,7 @@ public void validatePutWithString() {
                 .body("{\"name\":\"pallavisawale\",\"job\":\"SDET\"}")
```

Comment:
this is very primitive way to execute APIs, its always better to use requestSpecBuilder and responseSpecBuilder in restassured.

Reply:
Updated test cases..